src/AmazonPHP/SellingPartner/SellingPartnerSDK.php

```diff
@@ -84,6 +84,11 @@ public static function create(
         return new self($httpClient, $requestFactory, $streamFactory, $configuration, $logger);
     }
 
+    public function getConfiguration() : Configuration
```

Comment:
could you please rename this to `configuration(): Configuration` to follow the convention?

Reply:
Done...